src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml

```diff
@@ -19623,6 +19623,9 @@
   <LocaleResource Name="ShoppingCart.Rental.StartDateShouldBeFuture">
     <Value>Rental start date should be the future date</Value>
   </LocaleResource>
+  <LocaleResource Name="ShoppingCart.ReorderWarning">
+    <Value>Some products are not available anymore, so they weren't added to the cart.</Value>
```

Comment:
@mike-warren-dev You should also add this locale to the [migration file](https://github.com/nopSolutions/nopCommerce/blob/develop/src/Presentation/Nop.Web.Framework/Migrations/UpgradeTo470/LocalizationMigration.cs).